charts/cargohold/values.yaml

```diff
@@ -37,3 +37,6 @@ resources:
 #       domain: assets.example.com
 #       keyPairId: ABC....
 #       privateKeyPath: /etc/wire/cargohold/secrets/cf-pk.pem
+
+config:
```

Comment:
Maybe move this above the 'provide me please' block so it doesn't get lost in the shuffle.

Reply:
Done.